sdk/eventhub/event-hubs/src/impl/eventHubClient.ts

```diff
@@ -410,11 +410,10 @@ export class EventHubClient {
   private _createClientSpan(operationName: OperationNames, parentSpan?: Span | SpanContext): Span {
     const tracer = getTracer();
     const span = tracer.startSpan(`Azure.EventHubs.${operationName}`, {
-      kind: SpanKind.CLIENT,
+      kind: SpanKind.INTERNAL,
```

Comment:
How do we decide whether to use `CLIENT` vs `INTERNAL`?

Reply:
@lmolkova mentioned convenience calls are `INTERNAL` and API requests are `CLIENT` in #6310

Reply:
wait, for Event Hub - Send is a client span (there is no protocol-level span). Here is the issue for EventHub span kinds: https://github.com/Azure/azure-sdk-for-js/issues/6510

Reply:
This particular piece of code is for the following EvenHubClient operations: `getProperties`, `getPartitionIds`, and `getPartitionProperties`, which are REST calls to the service if I am not mistaken, not part of `send`?

Reply:
I talked to Chris and I guess I was confused because these do go over AMQP. The only one that is an actual convenience is `getPartitionIds` since that calls through to `getProperties`

Reply:
Updating to reflect this.